### PR TITLE
Test for class CountChar.

### PR DESCRIPTION
**Pull request by ricardo-ramos-moura**
### **Describe your change:**

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### **Checklist:**

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by aitorfi** [comment]
The changes you made to the DudeneyNumber class belong to a different PR (#3327) remove those and resolve the merge conflicts please.

**Review by aitorfi — CHANGES_REQUESTED** [review]
Remove the main method from the CountChar class as the test already demonstrates the algorithm works. Remove the changes made to the DudeneyNumber class as they belong to the PR #3327

**Review by aitorfi — CHANGES_REQUESTED** [review]
Remove the changes made to the DudeneyNumber class as they belong to the PR #3327 and resolve the merge conflicts please.

**Review by aitorfi — COMMENTED** [review]
LGTM

File: src/main/java/com/thealgorithms/maths/DudeneyNumber.java [hunk 1]
```diff
@@ -39,4 +39,4 @@ public static boolean isDudeney(int n) {
 
         return true;
     }
-}
+}
```
Comment (siriak) [inline src/main/java/com/thealgorithms/maths/DudeneyNumber.java:42]:
```suggestion
}

```

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**